packages/core/src/components/EditableTypography/EditableTypography.module.scss

```diff
@@ -1,11 +1,12 @@
 @import "~monday-ui-style/dist/mixins";
 
 .editableTypography {
-  display: inline-flex;
+  display: block;
   min-width: 0;
-  max-width: 100%;
   // Shifts the component to align the text with the container
   margin-left: -6px;
+  // Adds 6 px removed above to the element width to avoid unnecessary overflow
+  max-width: calc(100% + 6px);
```

Comment:
**question:** 
@talkor 

First of all, thank you very much for the quick feedback and pointing out accessibility issues.

I didn't go with "display: block" initially because it didn't fix the overflowing issue. I looked further and realized that what's causing this issue is this negative margin-left in combination with max-width set to 100%. Adding missing 6px to max-width fixes the issue without chaning the display value.

Wdyt about this approach to fixing this issue and do you think the display value still needs to be updated?

Reply:
Thanks @mashazyu ! Looks like a great solution. Since there's a change in the component we'll need to test it a bit to make sure it doesn't break anything. Assuming everything is ok I'll merge it. In the meantime I've added the hacktoberfest accepted tag. Thank you so much for this!